packages/core-ethereum/src/hashedSecret.ts

```diff
@@ -51,6 +48,25 @@ class HashedSecret {
     )
   }
 
+  public async hashFunction(msg: Uint8Array): Promise<Uint8Array> {
+    let toHash = msg
+
+    // @TODO Uncomment this to have salted hashes
+    // let toHash = u8aConcat(new TextEncoder().encode('HOPRnet'), await this.coreConnector.account.address, msg)
+
+    return (await this.coreConnector.utils.hash(toHash)).slice(0, HASHED_SECRET_WIDTH)
```

Comment:
is there a reason not to just import the hash function directly? Why do we need a coreConnector instance here?

Reply:
legacy code ...